discord/ext/pages/pagination.py

```diff
@@ -21,7 +21,10 @@
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 DEALINGS IN THE SOFTWARE.
 """
-from typing import Dict, List, Optional, Union
```

Comment:
Py-cord supports Python 3.8+ but typehinting without the `typing` module was introduced only in 3.9

Reply:
> Py-cord supports Python 3.8+ but typehinting without the `typing` module was introduced only in 3.9

hm @pre-commit-ci commited this after your commit @BobDotCom i dont know if this change should be rolled back in this case or just bot commit

Reply:
The `from __future__ import annotations` import fixes this. See [the python documentation](https://docs.python.org/3/library/__future__.html) and [my message in the discord server](https://canary.discord.com/channels/881207955029110855/1062602760899543070/1062800078277972058) for more info.